vox_nav_planner_server_rclcpp_node:
  ros__parameters:
    planner_name: "RRTConnect"
    planner_timeout: 600.0
    interpolation_parameter: 100                        # set to 0 if you wanna disable interpolating and smooting, otherwise 25 is a good default value                    
    robot_body_dimens:
      x: 0.6
      y: 0.6
      z: 0.6
    #robot_mesh_path: "package://botanbot_description/meshes/base_simplified.stl" # leave empty if you do not have one, robot_mesh_path: ""
    robot_mesh_path: ""

    # valid samples
    min_surfel_distance: 0.0
    max_surfel_distance: 0.2

    planner_type: "vox_nav_planning::ElevationPlanner"    # PRMstar: Reccomended
    se2_space: "SE2"                             # "DUBINS","REEDS", "SE2" ### PS. Use DUBINS OR REEDS for Ackermann
    rho: 2.0                                        # Curve radius for reeds and dubins only
    state_space_boundries: 
      minx: -50.0
      miny: -20.0
      maxx: 50.0
      maxy: 5.0
      minz: -1.0
      maxz:  1.0

    # exp
    # min_x: -20.0
    # min_y: -20.0
    # max_x: 50.0
    # max_y: 5.0

    # park
    # only used when clip map
    # min_x: -20.0
    # min_y: -140.0
    # max_x: 60.0
    # max_y: 13.0
    # min_z: -10.0
    # max_z: 10.0
